packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -715,7 +715,10 @@ - (NSMutableURLRequest *)_createRequest:(NSURL *)url headers:(NSDictionary * _Nu
 
 - (EXFileSystemSessionType)_importSessionType:(NSNumber * _Nullable)type
 {
-  return [type intValue] ?: EXFileSystemBackgroundSession;
+  if (type == nil) {
```

Comment:
I think it's more common to use `if (!type)` when checking for `nil`, but I think it's ok to leave this as it is 👍